test_data/makedata_suites/060_computed_values.js

```diff
@@ -0,0 +1,821 @@
+        `for doc in testView filter doc.cv_field == to_hex(doc.name) collect with count into c return c`,
+        `for doc in testView filter doc.cv_field == CONCAT('42_', TO_STRING(doc.field)) collect with count into c return c`,
+        `for doc in testView search doc.cv_field1=='foo' and doc.cv_field2=='bar' and doc.cv_field3=='baz' collect with count into c return c`,
+        `for doc in testView filter doc.cv_field == CONCAT(doc._key, ' ', doc._id, ' ', doc._rev) collect with count into c return c`,
+        `for doc in testViewV2 search doc.cv_field == SOUNDEX('sky') collect with count into c return c`, 
+        `for doc in testViewV2 search doc.cv_field == SOUNDEX('dog') collect with count into c return c`,
+        `for doc in testViewV2 search doc.cv_field_insert == SOUNDEX('frog') collect with count into c return c`,
+        `for doc in testViewV2 search doc.cv_field_update == SOUNDEX('beer') collect with count into c return c`,
+        `for doc in testViewV2 search doc.cv_field_replace == SOUNDEX('water') collect with count into c return c`,
+        `for doc in testViewV2 search doc.cv_field == null collect with count into c return c`,
+        `for doc in testViewV2 filter doc.cv_field == to_hex(doc.name) collect with count into c return c`,
+        `for doc in testViewV2 filter doc.cv_field == CONCAT('42_', TO_STRING(doc.field)) collect with count into c return c`,
+        `for doc in testViewV2 search doc.cv_field1=='foo' and doc.cv_field2=='bar' and doc.cv_field3=='baz' collect with count into c return c`,
+        `for doc in testViewV2 filter doc.cv_field == CONCAT(doc._key, ' ', doc._id, ' ', doc._rev) collect with count into c return c`
+      ]
+
+      let views_exp_output = [64000, 64000, 64000, 0, 0, 64000, 96000, 64000, 64000, 64000, 64000, 64000, 0, 0, 160000, 64000, 96000, 64000, 64000]
+
+      resultComparision(views_array, views_exp_output)
+
+
+      return 0;
+    },
+    clearDataDB: function (options, isCluster, isEnterprise, dbCount, database) {
+      print(`checking data ${dbCount}`);
+      let c1 = `c1_${dbCount}`;
+      let c2 = `c2_${dbCount}`;
+      let c3_insert = `c3_insert_${dbCount}`;
+      let c4_update = `c4_update_${dbCount}`;
+      let c5_replace = `c5_replace_${dbCount}`;
+      let c6_not_null = `c6_not_null_${dbCount}`;
+      let c7_hex = `c7_hex_${dbCount}`;
+      let c8_overwriteFalse = `c8_overwriteFalse_${dbCount}`;
+      let c9_overwriteTrue = `c9_overwriteTrue_${dbCount}`;
+      let c10_multiple = `c10_multiple_${dbCount}`;
+      let c11 = `c11_${dbCount}`;
+      let c12 = `c12_${dbCount}`;
+
+      Collection_array = [c1, c2, c3_insert, c4_update, c5_replace, c6_not_null, c7_hex, c8_overwriteFalse, c9_overwriteTrue, c10_multiple, c11, c12]
+
+      Collection_array.forEach(col => {
+        db.col.properties({computedValues: []})
+        //checking the properties set to null properly
+        if(db.col.properties()["computedValues"] === null){
```

Comment:
please add blanks around () and in }else{

Reply:
done.